cob_description/urdf/cob4_base/base.urdf.xacro

```diff
@@ -19,6 +19,9 @@
 
   <xacro:macro name="base" params="name use_old_joint_names">
 
+    <xacro:property name="drive_vel" value="19.95" />
+    <xacro:property name="steer_vel" value="117.81" />
```

Comment:
what are these numbers? rad per sec?
where do they come from?

so for cob4 base (which use the cob_wheel macro) these values are used
while for raw basee (which also use the cob_wheel macro) the default 100 is used (because the outer-scope properties are not set), correct?